### DISCUSSION TITLE
Integrating two scATAC + GEX datasets

### DISCUSSION THREAD

**Discussion by rjmusich** [satijalab/seurat#4810] [Q&A]
I have two scATAC + GEX datasets sequenced using the 10x Chromium Multiome protocol and processed through CellRanger-ARC. The datasets represent a control and disease state so it would be beneficial to be able to integrate the two datasets together and find differences between the two. I was hoping to be able to do something like creating a Seurat object for the control state (containing both the ATAC and GEX assays) and the disease state (containing both ATAC and GEX assays) and then find a way to merge the two objects together, but I have been having difficulties finding a vignette that does that. All of the vignettes I have come across seem to just be integrating an additional scATAC assay into a pre-existing Seurat object that already contains the ATAC and GEX assays from another dataset, but this would leave out the paired GEX assay for the added ATAC assay. What is the best way to go about processing this data? Am I able to create a Seurat object containing both the GEX and ATAC assay for the control state, then integrate the ATAC assay from the disease state in one step, then integrate the GEX assay from the disease state in a second step? I feel like I would be losing the paired information from the disease state if I were to analyze the data in this way.

**Comment by timoast** [comment]
If you run the `merge` function, it will merge all of the assays not just the active assay, so you won't lose any information. I would suggest the following:

1. Quantify the same set of peaks for the ATAC assay across the two different experiments (see https://satijalab.org/signac/articles/merging.html)
2. Make sure the assays that you want to merge have the same name in each object
3. Merge the different multiome objects using `merge()` 
4. Check whether there are batch effects between the datasets, and if so you can run a similar workflow except using `IntegrateData()` and `IntegrateEmbeddings()` rather than `merge()`

**Reply by rjmusich** [reply]
What I ended up doing is the following:
I performed QC independently on each of the 4 assays (diseased and control GEX and diseased and control ATAC). I filtered out the union of bad cells found in diseased GEX and diseased ATAC (because the cells should be paired so if a cell failed GEX QC but passed ATAC QC, it should still be removed),  then did the same for the control. I then just merged the diseased and control ATAC assays to create one ATAC object and merged the diseased and control GEX assays to create one GEX object. The ultimate goal of the analysis was mainly to perform DE between the diseased and control states so I then just performed DE analysis on the GEX and ATAC objects separately and compared results between the two sets. Does this analysis seem valid with what you suggested? Also, if I am interpreting your solution correctly, I should just be able to do a merge of the ATAC object and GEX object as my objects are now, correct?

**Reply by timoast** [reply]
Yes but you need to make sure the same peaks are used in both ATAC experiments
